src/components/NcRichContenteditable/docsHelpers.js

```diff
@@ -0,0 +1,95 @@
+export const userData = {
+	Test01: {
+		icon: 'icon-user',
+		id: 'Test01',
+		title: 'Test01',
+		source: 'users',
+		primary: true,
+	},
+	Test02: {
+		icon: 'icon-user',
+		id: 'Test02',
+		title: 'Test02',
+		source: 'users',
+		status: {
+			clearAt: null,
+			icon: '🎡',
+			message: 'Visiting London',
+			status: 'away',
+		},
+		subline: 'Visiting London',
+	},
+	'Test@User': {
+		icon: 'icon-user',
+		id: 'Test@User',
+		title: 'Test 03',
+		source: 'users',
+		status: {
+			clearAt: null,
+			icon: '🎡',
+			message: 'Having space in my name',
+			status: 'online',
+		},
+		subline: 'Visiting London',
+	},
+	'Test Offline': {
+		icon: 'icon-user',
+		id: 'Test Offline',
+		title: 'Test Offline',
+		source: 'users',
+		status: {
+			clearAt: null,
+			icon: null,
+			message: null,
+			status: 'offline',
+		},
+		subline: null,
+	},
+	'Test DND': {
+		icon: 'icon-user',
+		id: 'Test DND',
+		title: 'Test DND',
+		source: 'users',
+		status: {
+			clearAt: null,
+			icon: null,
+			message: 'Out sick',
+			status: 'dnd',
+		},
+		subline: 'Out sick',
+	},
+}
+
+export const userMentions = {
+	Test01: {
+		component: 'NcUserBubble',
+		props: {
+			displayName: 'Test01',
+		},
+	},
+	Test02: {
+		component: 'NcUserBubble',
+		props: {
+			displayName: 'Test02',
+		},
+	},
+	// FIXME mentions below don't show up in the NcRichText examples
```

Comment:
they need to be posted as `@"Test Offline"` etc.

Reply:
Seems like it, but the NcRichText doesn't render it with usual pattern
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/93392545/f3d39129-f9c1-478e-9504-cf665ce3238b)